rudder-lang/src/main.rs

```diff
@@ -103,10 +104,18 @@ fn add_file<'a>(
     path: &'a Path,
     filename: &'a str,
 ) -> Result<()> {
-    let content = fs::read_to_string(path)
-        .unwrap_or_else(|_| panic!("Something went wrong reading the file {}", filename));
-    let content_str = source_list.append(content);
-    past.add_file(filename, &content_str)
+    println!("|- {} {}", "Parsing".bright_green(), filename);
+    match fs::read_to_string(path) {
+        Ok(content) => {
+            let content_str = source_list.append(content);
+            past.add_file(filename, &content_str)
+        },
+        Err(e) => {
+            let tk = Token::new(filename, "");
+            println!("{:#?}", tk);
```

Comment:
The next step will be to make this kind of output only appear in a verbose mode as it will be mostly useless for users.

Reply:
This will likely be one of the next output formatting changes, but should go into another pull request. Use the `log` crate with a simple cli flag would be a good start.